.github/workflows/ci.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       AWS_DEFAULT_REGION: us-east-1
+      CLOUDFLARE_ACCOUNT_ID: 895762025d37fc687ecd72d7cc80204a
```

Comment:
Your account ID should probably go in GH secrets

Reply:
Why? It was always visible in `wrangler.toml` too. It isn't sensitive as you can not do anything without the `CLOUDFLARE_API_TOKEN`.

Reply:
Mhm, never noticed 🤔 I just like to hide those kind of details